libs/utils/utils.js

```diff
@@ -437,13 +467,22 @@ export function decorateAutoBlock(a) {
         a.target = '_blank';
         return false;
       }
-      if (key === 'fragment' && url.hash === '') {
-        const { parentElement } = a;
-        const { nodeName, innerHTML } = parentElement;
-        const noText = innerHTML === a.outerHTML;
-        if (noText && nodeName === 'P') {
-          const div = createTag('div', null, a);
-          parentElement.parentElement.replaceChild(div, parentElement);
+      if (key === 'fragment') {
+        if (url.hash === '') {
+          const { parentElement } = a;
+          const { nodeName, innerHTML } = parentElement;
+          const noText = innerHTML === a.outerHTML;
+          if (noText && nodeName === 'P') {
+            const div = createTag('div', null, a);
+            parentElement.parentElement.replaceChild(div, parentElement);
+          }
+        } else {
+          // Modals
```

Comment:
Will this change impact in page jump links (TOC)? Those aren't a modal or a fragment.

Reply:
This was a merge messup - will revert back to current utils.js code